src/pages/Home.tsx

```diff
@@ -68,16 +94,21 @@ function Home() {
     setFile(acceptedFiles[0]);
     const fr = new FileReader();
     fr.readAsText(acceptedFiles[0]);
-    fr.onload = () => console.log(fr.result);
+    //    fr.onload = () => console.log(fr.result);
```

Comment:
Can this be removed, or will we need it in the future?

Reply:
removed